Module2_4/Module2_4/StringParser.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+
+namespace Module2_4
+{
+    public class StringParser
```

Comment:
use just DataParser for example - StringParser means you will get string as result